package.json

```diff
@@ -1,20 +1,23 @@
 {
-  "name": "",
+  "name": "ReviewsModule",
   "version": "",
   "description": "This is the module on an ecommerce page that shows reviews of the store",
   "author": "",
   "license": "",
+  "main": "server/index.js",
   "engines": {
     "node": ">=6.13.0"
   },
   "scripts": {
+    "server": "nodemon server/index.js",
     "react-dev": "webpack -d --watch",
     "create-db": "mysql  --defaults-extra-file=./config.txt < schema.sql",
     "migrations": "node_modules/.bin/knex migrate:latest",
     "seed": "node_modules/.bin/knex seed:run",
     "seed-db": "npm run create-db && npm run migrations && npm run seed"
```

Comment:
interesting - I'm going to update my script to be more versatile like yours.

Reply:
I will probably adjust my script later to start webpack and nodemon at the same time as well!